lib/absinthe/utils/render.ex

```diff
@@ -48,7 +48,7 @@ defmodule Absinthe.Utils.Render do
   end
 
   defp escape_string(<<char::utf8, rest::binary>>, acc) do
-    escape_string(rest, [acc | <<char::utf8>>])
+    escape_string(rest, acc ++ [<<char::utf8>>])
   end
```

Comment:
This resolves the following error with `mix dialyzer` on Elixir 1.18:

```
lib/absinthe/utils/render.ex:51:5:improper_list_constr
List construction (cons) will produce an improper list, because its second argument is bitstring().
________________________________________________________________________________
done (warnings were emitted)
Halting VM with exit status 2
```